src/gui/mainwindow.ui

```diff
@@ -390,7 +390,7 @@
     <bool>true</bool>
    </property>
    <property name="text">
-    <string>&amp;Disabled</string>
+    <string>&amp;Do nothing</string>
```

Comment:
This is an option for "On Downloads Done" (other options are: Shutdown system, Hibernate System, etc.)